loopchain/baseservice/peer_manager.py

```diff
@@ -110,29 +139,64 @@ def get_peer_ids_hash(peer_ids):
         util.logger.debug(f"peer ids hash({peer_ids_hash})")
         return peer_ids_hash
 
-    @property
-    def peer_list(self) -> dict:
-        """
+    def load_peers_from_iiss(self):
+        util.logger.debug(f"load peers from iiss...")
 
-        :return: { peer_id:PeerInfo }
-        """
-        # util.logger.spam(f"peer_list({self.peer_list_data.peer_info_list})")
-        return self.peer_list_data.peer_info_list
+        request = {
+            "method": "ise_getPRepList"
+        }
 
-    @property
-    def peer_order_list(self) -> dict:
-        """
+        request = convert_params(request, ParamType.call)
+        stub = StubCollection().icon_score_stubs[ChannelProperty().name]
+        response = cast(dict, stub.sync_task().call(request))
+        response_to_json_query(response)
 
-        :return: { order:peer_id }
-        """
-        return self.peer_list_data.peer_order_list
+        util.logger.notice(f"in load_peers_from_iiss response({response})")
+        if not response['result'].get('preps'):
```

Comment:
If stub call failed and returned value created by `iconservice.icon_inner_service.MakeResponse.make_error_response`, 
returned json data may not have key named 
"reponse", which means could raise a KeyError exception.

Reply:
And.. if you intended in this line to check that `response['result'] has key `preps` inside, it is said that below is much pythonic way -
```
if 'preps' not in response['result']
```